docs/source/guides/network_node/staking_guide.rst

```diff
@@ -302,25 +311,27 @@ to indicate which are the automated rules (in our case, allowing listing of acco
 
 .. code:: bash
 
-    $ clef --keystore /path/to/keystore --chainid 5 --advanced --rules rules.js
+    $ clef --keystore <PATH TO KEYSTORE> --chainid <CHAIN ID> --advanced --rules rules.js
 
 
 
 Initialize a new stakeholder
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Before continuing with stake initiation and management, A setup step is required to configure nucypher for staking.
-This will create a configuration file (`~/.local/share/nucypher/stakeholder.josn`) containing editable configuration values.
+This will create a configuration file (`~/.local/share/nucypher/stakeholder.json`) containing editable configuration values.
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake init-stakeholder --provider <PROVIDER> --network <NETWORK_NAME>
+    (nucypher)$ nucypher stake init-stakeholder --signer <SIGNER URI> --provider <PROVIDER> --network <NETWORK_NAME>
+
+where:
 
-If you ran ``geth`` node as above, your ``<PROVIDER>`` is
-``ipc:///home/<username>/.ethereum/goerli/geth.ipc``
-(on MacOS, ``ipc:///Users/<username>/Library/Ethereum/goerli/geth.ipc``)
+    * If you utilized `Clef Setup`_, the ``SIGNER URI`` is ``clef:///home/<username>/.clef/clef.ipc``
+    * If you ran ``geth`` node as above, your ``<PROVIDER>`` is ``ipc:///home/<username>/.ethereum/geth.ipc``
+      (on MacOS for Goerli testnet, ``ipc:///Users/<username>/Library/Ethereum/goerli/geth.ipc``)
+    * ``<NETWORK_NAME>`` is the name of the NuCypher network domain where the staker will participate.
 
-``<NETWORK_NAME>`` is the name of the NuCypher network domain where the staker will participate.
 
 .. note:: If you're participating in NuCypher's incentivized testnet, this name is ``gemini``.
```

Comment:
Should we just delete any remaining references to *incentivized* testnet or CASI?

Reply:
✔️